### PR TITLE
Added extract_repo.py script for deploying open source infrastructure

### PR DESCRIPTION
**Pull request by eponomar**

### CODE REVIEW

File: common/extract_repo.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
+import sys
```
Comment (azhogov) [inline common/extract_repo.py]: add copyright
run pylint and add comments

Comment (eponomar) [reply common/extract_repo.py]: done

File: common/extract_repo.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2017 Intel Corporation
```
Comment (azhogov) [inline common/extract_repo.py]: 2018

Comment (eponomar) [reply common/extract_repo.py]: done

File: common/extract_repo.py [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+This module deploy infrastructure code on build\test machines
```
Comment (azhogov) [inline common/extract_repo.py]: This module deploys infrastructure code on build and test machines

**Pull request merged**